docs/source/markdown/podman-kube-play.1.md.in

```diff
@@ -122,6 +123,40 @@ spec:
 
 and as a result environment variable `FOO` will be set to `bar` for container `container-1`.
 
+`Kubernetes Secret`
+
+Kubernetes Secret represents a Podman named secret. The Kubernetes Secret is saved as a whole and may be referred as a source of environment variables or volumes in Pods or Deployments.
```

Comment:
```suggestion
Kubernetes Secret represents a Podman named secret. The Kubernetes Secret is saved as a whole and may be referred to as a source of environment variables or volumes in Pods or Deployments.
```
?

Reply:
@vrothberg is correct, the 'to' is needed there, seems like we've a grammar error that needs touching up elsewhere.  @ygalblum sorry about that, and would you mind fixing the other one while you're at it please?